lib/modulr/resources/base.rb

```diff
@@ -13,8 +13,9 @@ def initialize(response, attributes = {})
         end
       end
 
-      def self.map(original_attribute, mapped_attributes)
+      def self.map(original_attribute, mapped_attributes = nil)
         class_eval { attr_writer original_attribute.to_sym }
+        mapped_attributes ||= original_attribute
```

Comment:
Small ergonomic change to simplify the definition of some usual maps where the naming doesn't change from this:

```rb
map :status, :status
```

to just 

```rb
map :status
```